Chat/ClientApp/src/components/ChatThread.tsx

```diff
@@ -276,51 +334,7 @@ export default (props: ChatThreadProps): JSX.Element => {
           />
         )}
         <Ref innerRef={createdRef}>
-          <LiveAnnouncer>
-            <Chat
-              styles={chatStyle}
-              items={messagesWithAttached.map((message: any, index: number) => {
-                const liveAuthor = `${message.senderDisplayName} says `;
-                const messageContentItem = (
-                  <div>
-                    <LiveMessage message={`${message.mine ? '' : liveAuthor} ${message.content}`} aria-live="polite" />
-                    {renderHyperlink(message.content.message)}
-                  </div>
-                );
-                return {
-                  gutter: message.mine ? (
-                    ''
-                  ) : (
-                    <div
-                      className={messageAvatarContainerStyle(
-                        props.users[message.sender.communicationUserId] === undefined
-                          ? ''
-                          : props.users[message.sender.communicationUserId].emoji
-                      )}
-                    >
-                      {props.users[message.sender.communicationUserId] === undefined
-                        ? ''
-                        : props.users[message.sender.communicationUserId].emoji}
-                    </div>
-                  ),
-                  key: index,
-                  contentPosition: message.mine ? 'end' : 'start',
-                  message: (
-                    <Flex vAlign="end">
-                      <Chat.Message
-                        styles={chatMessageStyle(message.mine)}
-                        content={messageContentItem}
-                        author={message.senderDisplayName}
-                        mine={message.mine}
-                      />
-                      <div className={readReceiptIconStyle(message.mine)}>{readReceiptIcon(message)}</div>
-                    </Flex>
-                  ),
-                  attached: message.attached
-                };
-              })}
-            />
-          </LiveAnnouncer>
+          {props.messages.length > 0 ? chatComponent() : chatLoading()}
```

Comment:
Might not be safe to check against the message length. In the case of the very first time the length will always be zero.